src/app/main.js

```diff
@@ -61,6 +49,16 @@ class App {
             pushState: true
         });
 
+        function findAncestor(el, Element) {
```

Comment:
Hmm, maybe, although I'm not sure it would actually be used anywhere else (how often do you use `closest()`).